docs/user/openstack/install_upi.md

```diff
@@ -418,11 +407,11 @@ $ python -c '
 import yaml
 path = "install-config.yaml"
 data = yaml.safe_load(open(path))
-data["networking"]["networkType"] = "Kuryr"
+data["networking"]["networkType"] = "OpenShiftSDN"
 open(path, "w").write(yaml.dump(data, default_flow_style=False))'
 ```
 
-Also set `os_networking_type` to `Kuryr` in `inventory.yaml`.
+Also set `os_networking_type` to `OpenShiftSDN` in `inventory.yaml`.
```

Comment:
`os_networking_type` was only used to enable creation of openstack resources needed by Kuryr, it[ loos like](https://github.com/search?q=repo%3Aopenshift%2Finstaller%20os_networking_type&type=code) we can remove it from the inventory.